### ISSUE TITLE
Image blocks and blurring? (samples/simple-sphere.rb)

### ISSUE THREAD

**Issue by jasonrclark** [shoes/shoes4#1309]
Running through "bad" samples, and noticed that `samples/simple-sphere.rb` isn't close to running. It uses two big things that look problematic:

* `image` form that takes a block instead of pointing to a file
* `blur` method which influences the drawing

If SWT doesn't easily support `blur`, may consider just dropping support for it (or marking with a warning and not implementing?) Bears more research.

Image of app on Shoes 3:
![image](https://cloud.githubusercontent.com/assets/130504/20643126/59d1f176-b3d5-11e6-9767-83e7af0217d3.png)

**Comment by PragTob** [comment]
Not sure, there was some block form we said we're not gonna do... can't remember if it was image do or shape do.

**Comment by jasonrclark** [comment]
Yeah I recall the shape form too. Don't need a call right now but if we aren't doing it that same, cool as it is, should go.

**Comment by jasonrclark** [comment]
So looked into this from a couple different angles and haven't come to any satisfying conclusions:

1) There isn't an easy way to apply a `blur` effect that I can find in SWT. While certainly do-able with enough effort, it feels *way* out of proportion to my willingness to invest in it
2) Looking over shoes3 source, found there was also `glow` as another effect. Wonder if there are others :/
3) As far as `image` with block, it seems to share some similarity with `shape`--the basic art elements work within the block for both. `image` in Shoes 3 has `blur`/`glow` but doesn't support the positional `line_to` style methods. On the flip side, `shape` doesn't support `blur`/`glow` from what I can tell (behavior's odd, but doesn't seem to work).

So where's that leave us? I hate to leave something plainly incompatible, but this seems over the effort line to me. Given that, I'd propose to leave these lying for now (maybe bump this issue to 4.1 for later consideration if someone wants), but plumb in stub methods that warn about the lack of support. This keeps Shoes 3 apps that used that from crashing (which is a bad experience), but avoids sinking too much time and energy there now.

What do you think @PragTob?

**Comment by PragTob** [comment]
Sounds reasonable to me and I agree a lot - rather have a version out that is 95 to 99% compatible than quibble with the difficult stuff too long. :+1:
